packages/addons/addon-depends/snapcast-depends/asio/package.mk

```diff
@@ -10,4 +10,11 @@ PKG_URL="https://github.com/chriskohlhoff/asio/archive/asio-$PKG_VERSION.zip"
 PKG_SOURCE_DIR="asio-asio-$PKG_VERSION"
 PKG_DEPENDS_TARGET="toolchain"
 PKG_LONGDESC="Asio C++ Library."
-PKG_TOOLCHAIN="manual"
+PKG_TOOLCHAIN="autotools"
+PKG_BUILD_FLAGS="-sysroot"
+
+PKG_CONFIGURE_OPTS_TARGET="--without-boost --without-openssl"
+
+post_unpack() {
+  mv $PKG_BUILD/asio/* $PKG_BUILD
+}
```

Comment:
There already is `PKG_CONFIGURE_SCRIPT`, but that didn't work for some reason. It's been ~6 weeks and I don't recall why though :P

Reply:
Yes, in theory that should work but I don't think scripts/autoreconf will work when configure.ac is in a subdirectory - it assumes it will be in the root. So there'd need to be a bit of tweaking to make it work in a sub-directory, but it's probably not worthwhile right now.